### PR TITLE
Bug 1817466 - [pushapkscript] Remove unused pem files, configs for unused apps and aliases

### PR DESCRIPTION
**Pull request by gbrownmozilla**

### CODE REVIEW

**Comment by gbrownmozilla** [comment]
Not to land until https://github.com/mozilla-mobile/firefox-android/pull/1194 is on all branches.

File: pushapkscript/files/dep.pem
Comment (JohanLorenzo) [inline pushapkscript/files/dep.pem:1]: Good catch on this one. I assumed it was still used by Focus or Fenix, but it's not 👍

**Review by JohanLorenzo — COMMENTED** [review]
✅

**Comment by gbrownmozilla** [comment]
Ready to land now.

**Review by JohanLorenzo — APPROVED** [review]

**Pull request merged**